### PR TITLE
Level1 tsc

### PR DESCRIPTION
**Pull request by MaximKalinin**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Challenge 
+{
+    buzz(input: number): Promise<string | number> 
+    {
+        return new Promise(resolve => 
+            resolve(`${input % 3 === 0 ? "fizz" : ""}${input % 5 === 0 ? "buzz" : ""}` || input)
```
Comment (Pigotz) [inline src/index.ts:6]: :mage_man:

Comment (Pigotz) [reply src/index.ts:6]: You could have used `Promise.resolve(/* stuff */)` too :sunglasses:

**Review by Pigotz — APPROVED** [review]
:+1:

File: src/index.ts (hunk 1 above)
Comment (lurenzsgp) [reply src/index.ts:6]: Was the promise requested by the test of the exercise?

Comment (MaximKalinin) [reply src/index.ts:6]: Yes, it was

**Pull request closed without merging**